test/cri-containerd/runpodsandbox_test.go

```diff
@@ -946,21 +946,9 @@ func createSandboxContainerAndExecForCustomScratch(t *testing.T, annotations map
 func createContainerInSandbox(t *testing.T, client runtime.RuntimeServiceClient, ctx context.Context, podId, containerName, imageName string, command []string,
 	annotations map[string]string, mounts []*runtime.Mount, podConfig *runtime.PodSandboxConfig) string {
 
-	cRequest := &runtime.CreateContainerRequest{
-		Config: &runtime.ContainerConfig{
-			Metadata: &runtime.ContainerMetadata{
-				Name: containerName,
-			},
-			Image: &runtime.ImageSpec{
-				Image: imageName,
-			},
-			Command:     command,
-			Annotations: annotations,
-			Mounts:      mounts,
-		},
-		PodSandboxId:  podId,
-		SandboxConfig: podConfig,
-	}
+	cRequest := getCreateContainerRequest(podId, containerName, imageName, command, podConfig)
+	cRequest.Config.Annotations = annotations
+	cRequest.Config.Mounts = mounts
```

Comment:
Why did we change this in this PR?

Reply:
it wasn't really needed, I just added a utility function to  generate `CreateContainerRequest` with some defaults and used it here.